core/embed/rust/src/ui/model_tt/layout.rs

```diff
@@ -824,8 +819,11 @@ extern "C" fn new_confirm_total(n_args: usize, args: *const Obj, kwargs: *mut Ma
             paragraphs.add(Paragraph::new(&theme::TEXT_NORMAL, label));
             paragraphs.add(Paragraph::new(&theme::TEXT_MONO, value));
         }
-
-        let mut page = SwipeHoldPage::new(paragraphs.into_paragraphs(), theme::BG);
+        let mut page = if cancel_arrow {
+            SwipeHoldPage::with_cancel_arrow(paragraphs.into_paragraphs(), theme::BG)
+        } else {
+            SwipeHoldPage::new(paragraphs.into_paragraphs(), theme::BG)
+        };
```

Comment:
Right, that is because I had a bug there - `cancel_arrow` value was never read correctly. It should be set to `True` only for `Ethereum`

Fixed in [f9c618f](https://github.com/trezor/trezor-firmware/pull/3188/commits/f9c618f81c9d04d320fc2a3a3b41cda6e592af6d)